packages/story-editor/src/components/canvas/useInsertElement.js

```diff
@@ -47,7 +47,7 @@ function useInsertElement() {
   const {
     actions: { registerUsage },
   } = useMedia3pApi();
-  const { getProxiedUrl } = useCORSProxy();
+  const { getProxiedUrl, checkResourceAccess } = useCORSProxy();
```

Comment:
Note that lint error in this file now

Reply:
?

Reply:
There was a lint error in this file

https://github.com/google/web-stories-wp/runs/4266687792?check_suite_focus=true

You just fixed it in 76425572f524e1715ba3b89b9bfe40f85e767008